src/main/resources/application.yaml

```diff
@@ -36,13 +36,54 @@ spring:
 oidc:
   issuer: ${OIDC_ISSUER:http://fr-am:8080/openam/oauth2/hmcts}
 
+idam:
+  s2s-auth:
+    microservice: unspec_service
+
+unspecified:
+  response-pack-url: https://formfinder.hmctsformfinder.justice.gov.uk/n9-eng.pdf
+  system_update:
```

Comment:
Should we decide to stick to either dashes od underscores for config properties names (I think the general rule is to use dashes)? Wherever we have control over it obviously, because it differs in dependencies we use (e.g. for ccd client it needs to be core_case_data).

Reply:
will change to -